### ISSUE TITLE
More panics on decoding still images

### ISSUE THREAD

**Issue by Shnatsel** [image-rs/image-webp#33]
It seems that the AFL seeds for the WebP format are overly tuned to the libwebp implementation, and are not a good starting point for fuzzing other decoders.

When seeded with the smallest real-world images that I scraped from the web, the fuzzer still finds panics almost instantly: 
[smallest_scraped_webp.tar.gz](https://github.com/image-rs/image-webp/files/13779552/smallest_scraped_webp.tar.gz)

Sample image triggering a panic: [crash-70d04adad08d65334cc6e947843ae382420aef78.webp.gz](https://github.com/image-rs/image-webp/files/13779562/crash-70d04adad08d65334cc6e947843ae382420aef78.webp.gz)
which causes
```
thread '<unnamed>' panicked at image-webp/src/vp8.rs:851:14:
chunk size must be non-zero
```

**Comment by fintelia** [comment]
This crash is triggered by having a lossy image with width=0. Since zeroing out two bytes from the image header seems like the sort of thing the fuzzer should have been able to find pretty easily, I wonder if there just weren't any lossy images in the AFL seed corpus?

**Comment by Shnatsel** [comment]
There's more panics if you keep fuzzing with these seeds. For example, this file:
[crash-9ef3724ec5ce9543886897742c95cf6c497b87f7.webp.gz](https://github.com/image-rs/image-webp/files/13781718/crash-9ef3724ec5ce9543886897742c95cf6c497b87f7.webp.gz)
triggers this panic:
```
thread '<unnamed>' panicked at image-webp/src/lossless.rs:645:9:
assertion failed: num <= 16
```
Do you want me to file separate issues for these?

**Comment by fintelia** [comment]
That specific crash should be fixed by #35, but please do open issues if you find other crashes.
